package.json

```diff
@@ -26,6 +33,7 @@
     "gulp-streamify": "^1.0.2",
     "gulp-stylus": "^2.1.0",
     "gulp-uglify": "^1.5.1",
+    "react": "^0.14.7",
```

Comment:
For now the dev-dependency is following convention and laying the groundwork for the unit testing work I'm doing, which will require it. I completely agree that we should figure out the right way for our demos and applications to consume GeniBlocks along with the other required libraries (React, ReactDOM, Biologica, ...), but I don't think it needs to be part of this PR.